### PR TITLE
Improve format parameter doc in cli commands

### PR DESCRIPTION
**Pull request by ernilambar**
See #2837

### CODE REVIEW

File: php/commands/cli.php [hunk 1]
```diff
@@ -64,7 +64,12 @@ public function version() {
 	 * ## OPTIONS
 	 *
 	 * [--format=<format>]
-	 * : Accepted values: json
+	 * : Render output in a particular format.
+	 * ---
+	 * default: json
```
Comment (danielbachhuber) [inline php/commands/cli.php]: The default format isn't `json` in this case. We need some name for the default format, and we should make sure there's test coverage for the default format (as the build didn't fail when it should've).

File: php/commands/cli.php [hunk 2]
```diff
@@ -131,7 +136,15 @@ public function info( $_, $assoc_args ) {
 	 * : Limit the output to specific object fields. Defaults to version,update_type,package_url.
 	 *
 	 * [--format=<format>]
-	 * : Accepted values: table, csv, json, count. Default: table
+	 * : Render output in a particular format.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
```
Comment (danielbachhuber) [inline php/commands/cli.php]: We could add `yaml` to this set too.

**Comment by danielbachhuber** [comment]
@ernilambar Couple of things to finish up on this one.

File: php/commands/cli.php (hunk 1 above)
Comment (ernilambar) [inline php/commands/cli.php]: @danielbachhuber What name would you suggest? `list`?

Comment (danielbachhuber) [inline php/commands/cli.php]: Sure, `list` works for me.

File: features/cli-info.feature [hunk 3]
```diff
@@ -17,3 +17,9 @@ Feature: Review CLI information
       """
       {"wp_cli_packages_dir_path":"/tmp/wp-cli-home/.wp-cli/packages/"}
       """
+
+    When I run `wp cli info`
+    Then STDOUT should contain:
+      """
+      WP-CLI packages dir:    /tmp/wp-cli-home/.wp-cli/packages/
```
Comment (danielbachhuber) [inline features/cli-info.feature]: I think this is breaking on the tab character. You can instead check:

```
Then STDOUT should contain:
  """
  WP-CLI packages dir:
  """
```

**Pull request merged**